Pod/Classes/VOKManagedObjectMapper.m

```diff
@@ -156,6 +156,10 @@ - (id)checkString:(id)outputObject withDateFormatter:(NSDateFormatter *)dateForm
 - (id)checkClass:(id)inputObject managedObject:(NSManagedObject *)object key:(NSString *)key
 {
     Class expectedClass = [self expectedClassForObject:object andKey:key];
+    if (!expectedClass) {
+        VOK_CDLog(@"Failed to get class for %@\nProperty: %@", object, key);
+        return nil;
```

Comment:
Nope, that's in `- (Class)expectedClassForObject:(NSManagedObject *)object andKey:(id)key`